File: PersonnelManagement/PersonnelManagement/Front/classes/Requests_classes/Rank_respons.ts
import Person from '../../classes/person';
import Rank from '../../classes/rank';

export default class Rank_respons {
    constructor() {
        this.person = new Person();
        this.rank = new Rank();
        this.date_end = new Date();
    }
    person: Person;
    rank: Rank;
    date_end: Date;
}

File: PersonnelManagement/PersonnelManagement/Models/Rank_respons.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PersonnelManagement.Models
{
    public partial class Rank_respons
    {
        public Person Person { get; set; }
        public Rank Rank { get; set; }
        public DateTime Date_end { get; set; }
    }
}
